hbase-assembly/src/main/assembly/components.xml

```diff
@@ -168,5 +168,11 @@
       </includes>
       <fileMode>0644</fileMode>
     </fileSet>
+    <fileSet>
+      <directory>${project.build.directory}/maven-repo</directory>
+      <outputDirectory>hbase-webapps/maven-repo</outputDirectory>
```

Comment:
Should be `<outputDirectory>hbase-webapps/static/maven-repo</outputDirectory>` instead.